scipy/spatial/tests/test_distance.py

```diff
@@ -2246,6 +2246,9 @@ def test_jensenshannon():
                         [0.1954288, 0.1447697, 0.1138377, 0.0927636])
     assert_almost_equal(jensenshannon(a, b, axis=1),
                         [0.1402339, 0.0399106, 0.0201815])
+    assert_almost_equal(jensenshannon(np.array([0.027501475, 0.055202297], dtype='float32'),
```

Comment:
new tests should use `assert_allclose`
```suggestion
    assert_allclose(jensenshannon(np.array([0.027501475, 0.055202297], dtype='float32'),
```

Reply:
maybe worth changing the rest of this test at the same time for consistency

Reply:
Better to figure out the correct tolerances, maybe try `assert_allclose(..., rtol=1.5e-6)`. The default for `assert_allclose` is `rtol=1e-7`, while the default for `assert_array_almost_equal` is roughly `atol=1e-6`. The difference between `rtol` and `atol` can be quite significant.